editor/src/components/navigator/navigator-item/navigator-item.tsx

```diff
@@ -44,27 +49,16 @@ interface ComputedLook {
 
 export const BasePaddingUnit = 20
 
-export function getElementPadding(
-  elementPath: ElementPath,
-  visibleNavigatorTargets: Array<ElementPath>, // TODO: we only need this to filter out fragments, can be delete after 'Fragment support' FS is deleted
-): number {
-  if (isFeatureEnabled('Fragment support')) {
-    return EP.navigatorDepth(elementPath, visibleNavigatorTargets) * BasePaddingUnit
-  }
-
-  const ancestors = EP.getAncestors(elementPath)
-  const ancestorsNotInNavigator = ancestors.filter(
-    (path) => !visibleNavigatorTargets.some((navigatorPath) => EP.pathsEqual(path, navigatorPath)),
-  )
+export function getElementPadding(withNavigatorDepth: number): number {
   // an empty path and the storyboard is always part of the ancestorsNotInNavigator list and that doesn't matter,
   // so we can add 2 to the offset. NOTE: this 2 is also a constant in EP.navigatorDepth for the same reason
```

Comment:
this comment should probably be updated

Reply:
Good spot!

Reply:
Addressed in this PR: https://github.com/concrete-utopia/utopia/pull/3399